doc/crud.md

```diff
@@ -268,6 +362,11 @@ This is the simplest case as we don't need to worry about previous revisions.
     CDTDocumentRevision *saved = [datastore createDocumentFromRevision:rev
                                                                  error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc1")
+    rev.body = [....]
```

Comment:
A bit nit picky on my part but here you use 5 dots to represent the body content and 4 dots a few lines above (https://github.com/cloudant/CDTDatastore/pull/210/files#diff-d206a46a6112977c401e4345eb3e7414R353) and in the Objective C examples you consistently use 3.  I would say pick one representation (3 seems like the least effort) and use that throughout.